automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -288,4 +288,40 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     // Return the test result, or throw error
     testResult.get
   }
+
+  def verifyHailImport(notebookPage: NotebookPage, vcfPath: GcsPath, clusterName: ClusterName): Unit = {
```

Comment:
This might more appropriately belong to `ClusterMonitoringSpec` - there are a few other test-specific methods in `LeonardoTestUtils` that could be moved too.